tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -128,7 +128,7 @@ def image_compare(self, target, ref, diff, fuzz_value):
         out, err = p.communicate()
         p.wait()
 
-        if not (err == '0' and out == ''):
+        if not (err == '0\n' or err == '0'):
```

Comment:
This is same as two commits ago.  I have changed it mistakenly because I forgot about mac/linux differences.